docs/index.md

```diff
@@ -30,9 +30,16 @@ It can be used in high-performance async web frameworks, using either asyncio
 or trio, and is able to support making large numbers of concurrent requests.
 
 !!! note
-    The 0.8 release switched HTTPX into focusing exclusively on providing an async
-    client. It is possible that we'll look at re-introducing a sync API at a
-    later date.
+    HTTPX should currently be considered in alpha. We'd love early users and feedback,
+    but would strongly recommend pinning your dependencies to the latest median
+    release, so that you're able to properly review API changes between package
+    updates. Currently you should be using `http==0.8.*`.
```

Comment:
Small typo here: `httpx`

Reply:
And actually these also apply to the repo README.

Reply:
😬 *Thank you!*